hack/swagger-v2/tools.go

```diff
@@ -25,6 +25,8 @@ import (
 	"k8s.io/kube-openapi/pkg/common"
 	"k8s.io/kube-openapi/pkg/validation/spec"
 
+	_ "k8s.io/code-generator"
+
```

Comment:
I meant that we have main.go and tools.go, then we store this import to the tools.go.

Reply:
Oh, sure @tenzen-y.
Does it look good ?
https://github.com/kubeflow/training-operator/pull/2389/commits/c5eafbc6ddb337c88aeae13c15998236335e209a

Reply:
Yeah, looks great! Thank you1